RiotSharp/Http/Interfaces/IRateLimitedRequester.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace RiotSharp.Http.Interfaces
+{
+    public interface IRateLimitedRequester
+    {
+        /// <summary>
+        /// The number of allowed calls per 10 seconds
+        /// </summary>
+        int RateLimitPer10S { get; set; }
+        /// <summary>
+        /// The number of allowed calls per 10 minutes
+        /// </summary>
+        int RateLimitPer10M { get; set; }
```

Comment:
I forgot to add it there, but when I think about it now, I am not sure anymore if we should include it in the interface or not. Technically they are not required for the interface.